src/mono/mono/mini/CMakeLists.txt

```diff
@@ -45,78 +45,6 @@ if(HOST_WIN32)
   add_link_options(/OPT:NOICF)
 endif()
 
-# ICU
```

Comment:
If these are now unused, can we delete pkgconfig and brew related code from: https://github.com/dotnet/runtime/blob/0ede71f16e0f7595a6a46384df5b470019b89c7e/src/mono/CMakeLists.txt#L715-L787 and all the `PKG_CONFIG_PATH` lines in https://github.com/dotnet/runtime/blob/0ede71f16e0f7595a6a46384df5b470019b89c7e/src/mono/mono.proj?

Reply:
Thanks, will remove these codes.